### PR TITLE
veper

### PR DESCRIPTION
**Pull request by TsjipTsjip**
veper

image

### CODE REVIEW

**Comment by PolarTundra** [comment]
veper

File: Resources/Prototypes/_Umbra/Entities/Objects/Fun/toys.yml [hunk 1]
```diff
@@ -28,4 +28,35 @@
...
+  - type: EmitSoundOnTrigger
+    sound:
+      path: /Audio/Voice/Moth/moth_squeak.ogg
+  - type: MeleeWeapon
+    soundHit:
+      path: /Audio/Voice/Moth/moth_squeak.ogg
+  - type: Food
+    requiresSpecialDigestion: true
+    useSound:
+      path: /Audio/Voice/Moth/moth_scream.ogg
```
Comment (Simyon264) [inline Resources/Prototypes/_Umbra/Entities/Objects/Fun/toys.yml:62]: Duplicating the YAML like this looks a bit odd. You could make an entity `BasePlushieMoth` with the sounds and then parent of that. Just make sure `BasePlushieMoth` parents of `BasePlushie`.

**Comment by Necro-a** [comment]
at this rate can we just add all the moth plushies

**Comment by TsjipTsjip** [comment]
Should clarify, as I did not do this in the original PR for the meme, that I am making this PR on behalf of PolarTundra. This is their personal item.

**Review by Simyon264 — APPROVED** [review]
image
workz

**Pull request merged**